sechub-pds-solutions/multi/cluster_shared_volume.svg

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as my comment before - duplicated gfx should be avoided, "common" part necessary.